DOCS-421: base models v2 (#968)
viamrobotics/docs: docs/components/base/agilex-limo.md

---
title: "Configure a Agilex LIMO Base"
linkTitle: "agilex-limo"
weight: 40
type: "docs"
description: "Configure an Agilex LIMO base."
tags: ["base", "components"]
# SMEs: Matt Vella, Steve B
---

> A `agilex-limo` base is a model for [LIMO](https://global.agilex.ai/products/limo), a mobile robot designed around the AgileX mobile platform.

Reviewer: ```suggestion
The `agilex-limo` model supports [LIMO](https://global.agilex.ai/products/limo), a mobile robot designed around the AgileX mobile platform.
```
